tools/coding-style/include-linter.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
```

Comment:
Great job @xpol , I do like this idea to lint the include files. @ricardoquesada  Could you please have a look at this PR? thanks.